src/main/java/bowling/service/BowlingGameService.java

```diff
@@ -16,6 +17,10 @@ public BowlingGameService() {
     }
 
     public void startGame(Participant participant) {
+        Optional<Frames> frames = repository.findByParticipant(participant);
+        if (frames.isPresent()) {
+            throw new IllegalArgumentException("이미 등록한 참가자입니다.");
+        }
```

Comment:
```suggestion
        repository.findByParticipant(participant)
                .ifPresent(frames -> {
                    throw new IllegalArgumentException("이미 등록한 참가자입니다.");
                });
```
Optional은 isPresent()를 사용하지 않아도 될 만큼 다른 API들이 굉장히 좋습니다. 
isPresent()를 사용하고 싶다면 다른 API를 고려해보라는 신호라고 생각해보세요! 😄